gym_bridges/envs/bridges_env.py

```diff
@@ -261,11 +268,79 @@ def reset(self, state=None, gap_count=None):
 
         return self._state.copy()
 
+    def _draw_state(self) -> None:
+        block_size = renderer_config.BLOCK_SIZE * 0.5
+
+        x_window_coordinates = np.arange(
+            renderer_config.WINDOW_WIDTH // 2 - block_size * self.shape[1] // 2,
+            renderer_config.WINDOW_WIDTH // 2 + block_size * self.shape[1] // 2,
+            block_size,
+        )
+        y_window_coordinates = np.arange(
+            renderer_config.WINDOW_HEIGHT // 2 - block_size * self.shape[0] // 2,
+            renderer_config.WINDOW_HEIGHT // 2 + block_size * self.shape[0] // 2,
+            block_size,
+        )
+
+        for state_block, (y_window_coordinate, x_window_coordinate) in zip(
+            self._state.flatten(),
+            itertools.product(y_window_coordinates, x_window_coordinates),
+        ):
+            rect = pygame.Rect(
+                x_window_coordinate,
+                y_window_coordinate,
+                block_size,
+                block_size,
+            )
+            self._screen.blit(self._textures[state_block], rect)
+
+    def _initialize_pygame_if_necessary(self) -> None:
+        if self._initialize_pygame:
```

Comment:
optional (but preferred) nit: I  like the style of 

if not self._initialize_pygame:
  return

pygame.init() etc


benefits:
If it's self._initialize_pygame is false, it's very clear that we need not read further.

This also means the entire function isn't a nested if-statement.